### ISSUE TITLE
License seems to be incorrect

### ISSUE THREAD

**Issue by agnivade** [koddsson/travis-github-pr-bot#8]
Hi there,

I was planning to fork your code and therefore was checking out the license. The file has -
```
Copyright (c) 2012 Paul Bakaus, http://jqueryui.com/

This software consists of voluntary contributions made by many
individuals (AUTHORS.txt, http://jqueryui.com/about) For exact
contribution history, see the revision history and logs, available
at http://jquery-ui.googlecode.com/svn/
```

This does not seem to be right. Was there a copy-paste error somewhere ?

**Comment by koddsson** [comment]
Whoops. Let me change that real quick.

**Comment by koddsson** [comment]
Fixed in bbb45362ab92edb0c8970167efe1fa63a692bd36 😊
